game/templates/game/board.html

```
@@ -63,8 +87,9 @@
 
     <div class="promo-overlay active" id="welcomeOverlay" style="background: var(--page-bg); z-index: 10000;">
         <div class="promo-dialog">
-            <div style="margin-bottom: 20px; color:#f0c040;">
+            <div style="margin-bottom: 20px; color:#f0c040; position: relative;">
                 <a href="/" class="back-btn">← Back to Home</a>
+                <button id="themeToggle" class="theme-toggle" title="Toggle light / dark theme">🌙</button>
```

HTMLHint (1.9.2)

[warning] The type attribute must be present on <button> elements.

(button-type-require)